src/ui/components/VPNMainImage.qml

```diff
@@ -257,7 +314,6 @@ Rectangle {
         y: 5
         antialiasing: true
         smooth: true
-        color: showVPNOnIcon ? "#3FE1B0" : "#FF4F5E"
```

Comment:
I think removing this line means we'll need to set the value here for each state. This is what I'm seeing locally when starting up the app.
<img width="200" alt="Screen Shot 2021-11-16 at 2 03 53 PM" src="https://user-images.githubusercontent.com/22355127/142074586-5b26adf0-ed12-459a-94f8-a2816b56d41c.png">

Reply:
Your are totally right — I missed setting the value in the default state.